job_package_1/models/helper.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+
+def find_dynamic_tags(s, a, b): 
+  
+    # initially our result list is empty . 
+    res = list()
+
+    # find the length of given strings 
+    ls = len(s) 
+    la = len(a) 
+    lb = len(b)
+  
+    # currently make array and initially 
+    # put zero. 
+    x = [0] * ls
+    y = [0] * ls 
```

Comment:
Okay, this is an array, but what exactly will this array hold? think about that and change the variable names